lntest/harness.go

```diff
@@ -1991,6 +1991,24 @@ func (h *HarnessTest) CalculateTxFee(tx *wire.MsgTx) btcutil.Amount {
 	return balance
 }
 
+// CalculateTxWeight calculates the weight for a given tx.
+//
+// TODO(yy): use weight estimator to get more accurate result.
+func (h *HarnessTest) CalculateTxWeight(tx *wire.MsgTx) int64 {
```

Comment:
Do we need Calculate here? Why not just `TxWeight` and `TxFeeRate`?

Reply:
yeah it's a calculated value, wrapped inside `GetTransactionWeight`, which also justifies using `require.InEpsilonf` instead of `require.Equal` - if we can get the weight, we can do a precise equality check.

Reply:
I just usually find that `Calculate` is a redundant word when naming things. Not gonna die on the hill though. Seems fine to me.